roles/config-xfce/tasks/xfce.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: "Install additional packages for XFCE"
```

Comment:
Same here: need to support the right package manager, or error out if not supported:

```
TASK [config-xfce : Install additional packages for XFCE] ************************************************************
failed: [10.9.50.158] (item=[u'@Xfce Desktop']) => {"cmd": "dnf install -y python2-dnf", "failed": true, "item": ["@Xfce Desktop"], "msg": "[Errno 2] No such file or directory", "rc": 2}
	to retry, use: --limit @/home/esauer/src/infra-ansible/playbooks/bastion/install.retry
```